src/dune_rules/command.mli

```diff
@@ -63,6 +63,9 @@ module Args : sig
   (** Create dynamic command line arguments. *)
   val dyn : string list Action_builder.t -> _ t
 
+  (** Like [dyn] but output in a single argument, quoted and concatenated. *)
+  val dyn_as_single_arg : string list Action_builder.t -> _ t
```

Comment:
Not a fan of this function. The name is weird and the function's semantics are best understood just by reading the implementation. It's also barely useful as it's only used once in the code base.